<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="-KH-067-QA-02_DEMO2"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="27.393">
    <properties></properties>

            <testcase
                name="shoppingButtonInWishList"
                time="10.719"
                classname="WishListTest">

            <failure type="java.lang.AssertionError" message="expected [Prom.ua — маркетплейс Украины] but found [Кабінет покупця]">
                <![CDATA[java.lang.AssertionError: expected [Prom.ua — маркетплейс Украины] but found [Кабінет покупця]
                org.testng.Assert.fail(Assert.java:110)org.testng.Assert.failNotEquals(Assert.java:1413)org.testng.Assert.assertEqualsImpl(Assert.java:149)org.testng.Assert.assertEquals(Assert.java:131)org.testng.Assert.assertEquals(Assert.java:655)org.testng.Assert.assertEquals(Assert.java:665)WishListTest.shoppingButtonInWishList(WishListTest.java:39)java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)java.base/java.lang.reflect.Method.invoke(Method.java:577)org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)java.base/java.util.ArrayList.forEach(ArrayList.java:1511)org.testng.TestRunner.privateRun(TestRunner.java:806)org.testng.TestRunner.run(TestRunner.java:601)org.testng.SuiteRunner.runTest(SuiteRunner.java:433)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)org.testng.SuiteRunner.run(SuiteRunner.java:330)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)org.testng.TestNG.runSuitesLocally(TestNG.java:1176)org.testng.TestNG.runSuites(TestNG.java:1099)org.testng.TestNG.run(TestNG.java:1067)com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </failure>
    </testcase>


            <testcase
                name="addProductsToWishlist"
                time="9.674"
                classname="WishListTest">
            </testcase>
            <testcase
                name="goToWishlist"
                time="7.000"
                classname="WishListTest">
            </testcase>
</testsuite>
